### PR TITLE
Aut'akh Limb Crate (And Others)

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Adds a multitool to the engineering grasper, new sprites for the mining grasper, and most importantly a crate full of Aut'akh limbs to Robotics (at Jackboot's request)

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

**Comment by ParadoxSpace** [comment]
!review

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -706,3 +706,45 @@
...
+
+/obj/structure/closet/crate/autakh
+	name = "aut'akh crate"
+	desc = "Contains a number of limbs and augmentations created by the Aut'akh Commune."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "autakh_crate"
+	icon_opened = "autakh_crateopen"
+	icon_closed = "autakh_crate"
+
+/obj/structure/closet/crate/autakh/fill()
```
Comment (Arrow768) [inline code/game/objects/structures/crates_lockers/crates.dm]: Reduce that to two of each limb and one of each grasper.
Anything else that exceeds that should be ordered from cargo

Comment (ParadoxSpace) [reply code/game/objects/structures/crates_lockers/crates.dm]: fine

**Pull request merged**